### DISCUSSION TITLE
Header And Mapbox

### DISCUSSION THREAD

**Discussion by kagkouras** [firefly-iii/firefly-iii#6007] [General]
Hello ive got the latest version 5.7.2 and as i saw in the .env file theres not an option for the MAPBOX_API_KEY= ,also ive got a second question is it possible to change the header name?
![Screenshot_17](https://user-images.githubusercontent.com/13321214/163688334-533e41bd-c22b-4316-8e1c-18f527082fe8.png)

**Comment by JC5** [comment]
You can add the key yourself if you want.

What header do you want to change?
